spring-cloud-loadbalancer/src/main/java/org/springframework/cloud/client/loadbalancer/impl/annotation/LoadBalancerClients.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2013-2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.client.loadbalancer.impl.annotation;
+
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Import;
+
+/**
+ * Convenience annotation that allows user to combine multiple <code>@LoadBalancerClient</code>
+ * annotations on a single class (including in Java 7).
+ *
+ * @author Dave Syer
+ */
+@Configuration
+@Retention(RetentionPolicy.RUNTIME)
+@Target({ ElementType.TYPE })
+@Documented
+@Import(LoadBalancerClientConfigurationRegistrar.class)
+public @interface LoadBalancerClients {
+
+	LoadBalancerClient[] value() default {};
+
+	Class<?>[] defaultConfiguration() default {};
```

Comment:
maybe a see javadoc tag for LoadBalancerClient.configuration if that's what's implied